src/app/index.js

```diff
@@ -40,6 +81,9 @@ angular.module('app', [
 
     // If no states were matched, redirect to home
     $urlRouterProvider.otherwise('home')
+
+    $authProvider.baseUrl = 'http://localhost:3000/api'
```

Comment:
Yeah this PR should be merged first. It was hard to avoid things like this when PRs sit in limbo, so we might end up with some merge conflicts, but they won't be bad to fix. :)